lusidtools/cocoon/seed_sample_data.py

```diff
@@ -0,0 +1,99 @@
+import pandas as pd
+from lusidtools.cocoon.cocoon import load_from_data_frame
+from lusidtools.cocoon.utilities import load_json_file
+import logging
+from pathlib import Path
+
+logger = logging.getLogger()
+
+mappings = dict(load_json_file("config/seed_sample_data.json"))
+
+
+def seed_data(
+    api_factory,
+    domains,
+    scope: str,
+    transaction_file: str,
+    file_type: str,
+    mappings: dict = mappings,
+    sub_holding_keys=[],
+):
+
+    """
+    Description:
+    This function allows users to seed their LUSID environment with some core data (e.g. instruments,
+    portfolios, and transaction) from one file.
+
+    :param lusid.utilities.ApiClientFactory api_factory: The api factory to use.
+    :param list[str] domains: A list of the file_types for upload.
+    :param str scope: The scope of the transaction portfolio.
+    :param str transaction_file: a source file of transaction data.
```

Comment:
It can be either.
The file ultimately get passed to pandas.from_csv which accepts either absolute or relative. 
I have added tests for both absolute and relative files paths to the unit tests.